Main.sublime-menu

```diff
@@ -1,108 +1,103 @@
 [
   {
     "id": "selection",
-    "children":
-    [
+    "children": [
       {
         "caption": "MultiEditUtils",
-        "children":[
-          { "command": "jump_to_last_region" },
-          { "command": "add_last_selection" },
-          { "command": "cycle_through_regions" },
-          { "command": "normalize_region_ends" },
-          { "command": "split_selection" },
-          { "command": "strip_selection"},
-          { "command": "remove_empty_regions"},
-          { "command": "multi_find_all"},
-          { "command": "selection_fields", "caption": "Selection as Fields", "args": {"mode": "toggle"} },
-          { "command": "preserve_case"}
+        "children": [
+          {
+            "command": "jump_to_last_region"
+          },
+          {
+            "command": "add_last_selection"
+          },
+          {
+            "command": "cycle_through_regions"
+          },
+          {
+            "command": "normalize_region_ends"
+          },
+          {
+            "command": "split_selection"
+          },
+          {
+            "command": "strip_selection"
+          },
+          {
+            "command": "remove_empty_regions"
+          },
+          {
+            "command": "multi_find_all"
+          },
+          {
+            "command": "selection_fields",
+            "caption": "Selection as Fields",
+            "args": {
+              "mode": "toggle"
+            }
+          },
+          {
+            "command": "preserve_case"
+          },
+          {
+            "caption": "Expand Selection to …",
+            "command": "jump_to_interactive",
+            "args": {
+              "extend": true
+            }
+          }
         ]
       }
     ]
   },
+  {
+    "id": "goto",
+    "children": [
+      {
+        "caption": "Jump to …",
+        "command": "jump_to_interactive"
+      }
+    ]
+  },
   {
     "caption": "Preferences",

```

Comment:
please also remove overrides for caption and mnemonic, simply targetting by id for existing menu tree entries should suffice